PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobDeployFactory.java

```diff
@@ -111,6 +110,16 @@ public Deployable buildSparkBatchJobDeploy(@NotNull SparkSubmitModel submitModel
 
                     if (storageAccount.getAccountType() == StorageAccountType.ADLSGen2) {
                         destinationRootPath = submitModel.getJobUploadStorageModel().getUploadPath();
```

Comment:
Maybe we can get the root path by `clusterDetail.getDefaultStorageRootPath()`

Reply:
for gen2, it will return format like abfs://xx@xx.xx , still need to convert to format https://xx.xx.xx